This patch increases the maximum number of rows returned by a SPARQL
query over HTTP.

It follows the advice found here:
https://github.com/openlink/virtuoso-opensource/issues/700#issuecomment-351846621

Patch by Roel Janssen <roel@gnu.org>

diff -crB virtuoso-opensource-7.2.5-orig/libsrc/Wi/sparql_io.sql virtuoso-opensource-7.2.5/libsrc/Wi/sparql_io.sql
*** a/libsrc/Wi/sparql_io.sql	2020-01-17 15:21:32.203935981 +0100
--- b/libsrc/Wi/sparql_io.sql	2020-01-17 15:25:14.804234759 +0100
***************
*** 3251,3257 ****
    named_graphs := vector ();
    using_graphs := vector ();
    using_named_graphs := vector ();
!   maxrows := 1024*1024; -- More than enough for web-interface.
    deadl := 0;
    http_meth := http_request_get ('REQUEST_METHOD');
    ini_dflt_graph := virtuoso_ini_item_value ('SPARQL', 'DefaultGraph');
--- 3251,3257 ----
    named_graphs := vector ();
    using_graphs := vector ();
    using_named_graphs := vector ();
!   maxrows := ((16*1024*1024)/8)-2; -- More than enough for web-interface.
    deadl := 0;
    http_meth := http_request_get ('REQUEST_METHOD');
    ini_dflt_graph := virtuoso_ini_item_value ('SPARQL', 'DefaultGraph');
